otterdog/eclipse-pullpiri.jsonnet

```diff
@@ -4,10 +4,36 @@ orgs.newOrg('eclipse-pullpiri') {
   settings+: {
     description: "",
     name: "Eclipse Pullpiri",
+    packages_containers_internal: false,
+    packages_containers_public: false,
```

Comment:
`If members can publish public (or internal) releases / containers visible to anyone`

Here, the scope of member is not clear, so `false` is entered for now.

Reply:
so members refers to any committer / project lead of the project. As an open-source project it would be natural to publish public releases / containers visible to anyone I would assume.

Reply:
I agree with your comment. I removed the relevant part and committed the update.